src/app/components/FilterProducts/FilterProducts.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Button,
+  Menu,
+  MenuItem,
+  ListItemIcon,
+  Theme,
+  Box,
+} from '@mui/material';
+import { ExpandMore, ExpandLess, Sort } from '@mui/icons-material';
+import { useProducts } from '@/services/productService';
+import { mapFromCMSProductToProduct } from '@/app/helpers';
+
+const ThemeValues = {
+  values: {
```

Comment:
theme values to raczej nie tutaj :)

Reply:
a to tutaj dodane do zgodności typów, mam tak w większości komponentów :)